src/main/java/com/couchbase/client/dcp/config/ClientEnvironment.java

```diff
@@ -235,6 +251,11 @@ public Builder setPassword(String password) {
             return this;
         }
 
+        public Builder setOpenStreamTimeout(long openStreamTimeout) {
```

Comment:
maybe startStream timeout since thats how the method is called on the client api?